django-pagination/Readme.md

```diff
@@ -0,0 +1,77 @@
+# Django Project: Python Wiki
```

Comment:
:warning: **Fact Check**

It looks like the additional `Readme.md` file is redundant as having the default `README.md` should be sufficient.